<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pub</class>
 <widget class="QWidget" name="pub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{

background-color:transparent;
color: #aa54ff; 
}
#filter,#mediaplayer,#notif,#pdf{
background-color: transparent;
}


#pub{
background-color: #040f13;
}
#sidemenu{
background-color: #071e26; 
border-radius: 20px; 
}
#ajouter 
{
padding: 20px; 
background-color: #7f3fbf; 
border-radius: 5px;
color : #fff;
}
QPushButton,QLineEdit{
padding: 20px; 
background-color: #040f13; 
border-radius: 5px; }
#header{
background-color: #071e26; 
border-radius: 10px;
}
#rightsidemenu
{
background-color: #071e26; 
border-radius: 20px; 
}
#rightsidemenu_2
{
background-color: #071e26; 
border-radius: 20px; 
}

#mainbody{
background-color: #071e26; 
border-radius: 10px;
}</string>
  </property>
  <widget class="QFrame" name="pdf">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>535</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>50</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5"/>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>9</y>
      <width>72</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>20</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4"/>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>103</y>
      <width>1001</width>
      <height>561</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="sidemenu">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>191</width>
       <height>391</height>
      </rect>
     </property>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>113</width>
        <height>188</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>  encadrant</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>111</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>etudiant</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>111</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>materiel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>111</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>sponsor</string>
       </property>
      </widget>
      <widget class="QLabel" name="student_pic">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons/student.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="materiel_pic">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons/flask.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="sponsor_pic">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons/handshake.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="encadrant_pic">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons/teacher (1).png</pixmap>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Publication</string>
       </property>
      </widget>
      <widget class="QLabel" name="research_pic">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons/research.png</pixmap>
       </property>
      </widget>
      <zorder>pushButton_2</zorder>
      <zorder>pushButton_3</zorder>
      <zorder>pushButton_5</zorder>
      <zorder>pushButton_4</zorder>
      <zorder>student_pic</zorder>
      <zorder>materiel_pic</zorder>
      <zorder>sponsor_pic</zorder>
      <zorder>encadrant_pic</zorder>
      <zorder>pushButton_6</zorder>
      <zorder>research_pic</zorder>
     </widget>
     <widget class="QPushButton" name="tri_pic_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri Desc</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/funnel.png</normaloff>icons/funnel.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri Pub</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/funnel.png</normaloff>icons/funnel.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>on</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>off</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>auto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_pic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri ASC</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/funnel.png</normaloff>icons/funnel.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="mainbody">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>400</width>
       <height>381</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>50</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>9000</width>
       <height>9000</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Liste des publications</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLabel" name="recherche_pic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>20</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>icons/search (1).png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="filter">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_delete">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/delete.png</normaloff>icons/delete.png</iconset>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>341</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_update">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>220</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/edit.png</normaloff>icons/edit.png</iconset>
      </property>
     </widget>
     <zorder>filter</zorder>
     <zorder>label_2</zorder>
     <zorder>lineEdit_4</zorder>
     <zorder>recherche_pic</zorder>
     <zorder>btn_delete</zorder>
     <zorder>tableView</zorder>
     <zorder>btn_update</zorder>
    </widget>
    <widget class="QLabel" name="popupLabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>350</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>40</y>
       <width>311</width>
       <height>381</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QFrame" name="rightsidemenu">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>601</width>
         <height>591</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>271</width>
          <height>161</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="nombrepub">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>nobr pub</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_id_create">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>ID_publication</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_sujet_create">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Sujet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_propriete_create">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Propriété</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_description_create">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Description</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter une publication</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_create">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter_btn">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>300</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>350</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/pdf.png</normaloff>icons/pdf.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Excel</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/excel.png</normaloff>icons/excel.png</iconset>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QFrame" name="rightsidemenu_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>301</width>
         <height>411</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>281</width>
          <height>211</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLineEdit" name="lineEdit_nbpub_modifier">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>nobr pub</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_id_modifier">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>ID_publication</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_sujet_modifier">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Sujet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_propriete_modifier">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Propriété</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_modifier"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_description_modifier">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Description</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="modifier">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>310</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier une publication</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier_annuler">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>310</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>annuler </string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>961</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>42</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_13">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/hamburger.png</normaloff>icons/hamburger.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>50</horstretch>
         <verstretch>50</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string>InteLab</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>statistique</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/statistic.png</normaloff>icons/statistic.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
